templates/focus-ios.xliff

```diff
@@ -1,36 +1,62 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 http://docs.oasis-open.org/xliff/v1.2/os/xliff-core-1.2-strict.xsd">
-  <file original="Blockzilla/Info.plist" source-language="en" datatype="plaintext">
+  <file original="Blockzilla/InfoPlist.strings" datatype="plaintext" source-language="en">
```

Comment:
Since the file name has changed here, from Info.plist to InfoPlist.strings, it invalidates all existing localizations contained within that file section, unfortunately.